src/shared/lib/HotspotSet.ts

```diff
@@ -11,9 +11,9 @@
  */
 export default class HotspotSet {
```

Comment:
We should consider renaming this class to something more generic. Maybe in another PR...